.devcontainer/Dockerfile

```diff
@@ -8,9 +8,29 @@
 
 FROM ghcr.io/cse-labs/k3d:latest
 
+# RUN apk update
```

Comment:
Let's remove commented commands for a clean looking Dockerfile.

Reply:
Great catch; I have removed all technical debt that I saw in the middle.  I left the comments on chekov, as they seem to be more informative.